docs/BENCH-CONFIG-SPEC.md

```diff
@@ -0,0 +1,104 @@
+# Benchmarking config specification
+
+## Config structure
+
+Benchmarking config files are written in JSON format and have few reserved keys:
+ - `PARAMETERS_SETS` with benchmark parameters in each set
+ - `TEMPLATES` map where each named instance combines parameters sets and template-specific parameters
+ - `INCLUDE` list which specifies parameters sets to include in template
+
+Formatting example:
+```json
+{
+    "PARAMETERS_SETS": {
+        "parameters_set_name_0": Dict or List[Dict] of any JSON-serializable with any level of nesting,
+        ...
+    },
+    "TEMPLATES": {
+        "template_name_0": {
+            "INCLUDE": ["parameters_set_name_0", ...],
+            Dict of any JSON-serializable with any level of nesting overwriting parameter sets
+        },
+        ...
```

Comment:
Please fix it.

Reply:
It's expected to have broken highlighting because this template is not with correct JSON format.